qutebrowser/misc/sessions.py

```diff
@@ -411,17 +411,35 @@ def _load_window(self, win):
         window.show()
         tabbed_browser = objreg.get('tabbed-browser', scope='window',
                                     window=window.win_id)
-        tab_to_focus = None
-        for i, tab in enumerate(win['tabs']):
-            new_tab = tabbed_browser.tabopen(background=False)
+        # Put the active tab in index 0 to start with because the first tab
+        # added to a QTabWidget will always be shown so we don't want that to
+        # be a background tab.
```

Comment:
I don't understand this. I wrote a small test program:

```python
from PyQt5.QtWidgets import QApplication, QWidget, QTabWidget


class Widget(QWidget):

    def __init__(self, idx, parent=None):
        super().__init__(parent)
        self.idx = idx

    def showEvent(self, e):
        super().showEvent(e)
        print(f"shown {self.idx}")

    def hideEvent(self, e):
        super().hideEvent(e)
        print(f"hidden {self.idx}")


app = QApplication([])
w1 = Widget(1)
w2 = Widget(2)
w3 = Widget(3)

tw = QTabWidget()
tw.addTab(w1, "1")
tw.addTab(w2, "2")
tw.addTab(w3, "3")
tw.setCurrentIndex(1)

tw.show()

app.exec_()
```

and only tab 2 gets its `showEvent` called, as you'd expect. @blueyed are you sure this is necessary?

Reply:
@The-Compiler 
The code is not from me - I've just created the PR from an initial patch/branch.

Let me know if you mean something else, but I am using this myself.  The last time I've checked master/release would not work (i.e. it is necessary in general).

btw: please check if it has an influence on private windows - for me I get no content for rendered pages there (!), but have not checked without this branch / on master/release yet.

Reply:
I didn't note down any differences between versions so I don't know if there where different behaviours between versions but I do remember it happening on 5.13.0-3 which was in kde-unstable at the time. Without doing this then when restoring a session with lazy loading enabled the first tab and the one that was active when the session was saved would be loaded. With this only the previously active tab would be loaded. I don't remember if I reproduced it with a minimal example or not. Maybe I missattributed the behaviour.